compatibility-tests/multiclient/src/compatibility-test/java/tech/pegasys/artemis/compatibility/multiclient/StatusMessageCompatibilityTest.java

```diff
@@ -50,7 +49,6 @@ public void tearDown() {
   }
 
   @Test
-  @Disabled("Prysm doesn't prefix responses with the status code")
```

Comment:
Not sure that it matters, but it does feel a bit funny for this to be included in this changeset.

Reply:
It could be a separate PR, but this is the only compat test we currently have and the aim is to run it (it actually does pass now so probably should have been enabled in an earlier PR).